src/app.rs

```diff
@@ -484,14 +485,26 @@ where
             info!("Rustc git commit: {}", hash);
         }
 
+        let _thread_count :usize = env::var("AMETHYST_NUM_THREADS")
```

Comment:
Remove the underscore from this variable name.

Reply:
And done! For some reason rust compiler suggested that.